asminc/cbm_kernal.inc

```diff
@@ -100,14 +100,26 @@ UDTIM          := $FFEA
 
 ; ---------------------------------------------------------------------------
 ; Kernal routines, direct entries
+;
+; Unlike the above, these are not standard functions with entries in the jump
+; table. They do not exist in all Kernals, and where they do the entry point is
+; specific to that particular machine and possibly even Kernal version.
+;
+; This is list is not comprehensive: missing items for particular machines
```

Comment:
The first "is" is superfluous.

Reply:
Thanks. I've fixed this and force-pushed the new commits.